NGitLab/Models/Permissions.cs

```diff
@@ -0,0 +1,13 @@
+using System.Text.Json.Serialization;
+
+namespace NGitLab.Models
+{
+    public class Permissions
```

Comment:
Non generic objects related to Project are prefix with it (ProjectStatistics).

Can we keep the same principle for permission(s)? As it might be generic enough to collide with something else at some point

Reply:
Sure!